app/src/ui/toolbar/branch-dropdown.tsx

```diff
@@ -276,6 +276,8 @@ export class BranchDropdown extends React.Component<
       return null
     }
 
+    this.props.dispatcher.recordCheckRunsPopoverOpened()
```

Comment:
This is a render method, so might be invoked many times after the popover is opened. Maybe this should be in `AppStore::_setShowCIStatusPopover` method when `showCIStatusPopover` is `true`?

Reply:
Good point!